### ISSUE TITLE
Importing CSS from node_modules fails to parse CSS correct

### ISSUE THREAD

**Issue by joms** [remix-run/remix#108]
Importing CSS from node_modules to use in `LinksFunction` fails.

Example:
```
import tabsStyle from "@reach/tabs/style.css";

export const links: LinksFunction = () => {
    return [{ rel: "stylesheet", href: tabsStyle }];
};
```

This results in the error

```
/Users/a504767/code/hly-remix/node_modules/@reach/dialog/styles.css:4
:root {
^

SyntaxError: Unexpected token ':'
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/a504767/code/hly-remix/build/index.js:98:32)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
```

However importing relative to directory works:

```
import tabsStyle from "../node_modules/@reach/tabs/style.css";
```

MacOS Big Sur
Node 14.15.4
Remix 0.17

**Comment by 3lang3** [comment]
same issue #959

**Comment by 3lang3** [comment]
> Looks like this might be fixed in 0.18? [1235a3b#diff-34cad44ad43b66868436303096f3a3b58a25f9ca83ea0865fe49e6eb16fdc98cR34](https://github.com/remix-run/remix/commit/1235a3b5b5b5fd421aeb8a56086121d7a713316b#diff-34cad44ad43b66868436303096f3a3b58a25f9ca83ea0865fe49e6eb16fdc98cR34)

`v1.0.6` There's still a problem.

**Comment by ryanflorence** [comment]
This works now (#959 is a different thing)
